records_mover/records/delimited/conversions.py

```diff
@@ -39,13 +39,17 @@
 # account for MM/DD time.
 #
 # https://github.com/bluelabsio/records-mover/issues/75
-#
+
+# TODO: Need to make these functions or this will be very brittle.
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker

Reply:
Moved to Asana.